README.md

```diff
@@ -79,6 +101,13 @@ RESCRIPt is released under a BSD-3-Clause license. See LICENSE for more details.
 
 However, other resources accessible via RESCRIPt are released under different licenses, as detailed below.
 
-**The SILVA database** versions are released under different licenses. Refer to the [current SILVA release license information](https://www.arb-silva.de/silva-license-information/) for more details.
+**If using the SILVA database** (*e.g.*, with `get-silva-data`): Versions are released under different licenses. Refer to the [current SILVA release license information](https://www.arb-silva.de/silva-license-information/) for more details.
+
+**If using NCBI Genbank data** (*e.g.*, with `get-ncbi-data`): See the [NCBI disclaimer and copyright notice](https://www.ncbi.nlm.nih.gov/home/about/policies/) for more details.
+
+**If using GTDB data** (*e.g.*, with `get-gtdb-data`): See the [GTDB "about" page](https://gtdb.ecogenomic.org/about) for more details.
+
+**If using UNITE data** (*e.g.*, with `get-unite-data`): See the [UNITE citation page](https://unite.ut.ee/cite.php) for more details.
+
+**If importing and processing RDP data**: See both the [RDP Staff GitHub page](https://github.com/rdpstaff) and the [RDP SourceForge page](https://sourceforge.net/projects/rdp-classifier/) for more details.
```

Comment:
I would leave out this line and put it in the tutorial instead (if it is not already there).